src/openzaak/components/catalogi/tests/test_roltype.py

```diff
@@ -353,6 +353,86 @@ def test_filter_geldigheid(self):
         self.assertEqual(len(data), 1)
         self.assertEqual(data[0]["url"], f"http://testserver{reverse(roltype)}")
 
+    def test_filter_roltype_omschrijving(self):
+        rt1 = RolTypeFactory.create(
+            omschrijving="Behandelaar van klacht", zaaktype__concept=False
+        )
+        rt2 = RolTypeFactory.create(
+            omschrijving="Verwerker van aanvraag", zaaktype__concept=False
+        )
+        rt3 = RolTypeFactory.create(
+            omschrijving="Behandelaar van verzoek", zaaktype__concept=False
+        )
+
+        roltype_list_url = reverse("roltype-list")
+
+        response = self.client.get(roltype_list_url, {"omschrijving": "behandelaar"})
+
+        self.assertEqual(response.status_code, 200)
+
+        data = response.json()["results"]
+
+        self.assertEqual(len(data), 2)
+
+        omschrijving_1_url = reverse("roltype-detail", kwargs={"uuid": rt1.uuid})
+        omschrijving_3_url = reverse("roltype-detail", kwargs={"uuid": rt3.uuid})
```

Comment:
With `reverse` from `vng_api_common` you can pass the object itself, which is slightly less verbose
```suggestion
        omschrijving_1_url = reverse(rt1)
        omschrijving_3_url = reverse(rt3)
```